src/content/api/loaders.md

```diff
@@ -362,7 +386,9 @@ Resolves the given request to a module, applies all configured loaders and calls
 ### `this.resolve`
 
 ``` typescript
+
```

Comment:
is this spacing required by the linter?

Reply:
Yes it will report the error: `./src/content/api/loaders.md: 388: MD031 Fenced code blocks should be surrounded by blank lines`